### ISSUE TITLE
Not able to resume the test accuracy after loading the pretrain model

### ISSUE THREAD

**Issue by Ldpe2G** [dmlc/mxnet#3206]
I am using the lstm for some classification tasks, but met a strange problem.
For example, 
after the last training step, I tested the model on the test set, the accuracy is 96%, and I save the
model. Then I want to resume training, I load the pretrain model and copy the pretrain parameters to the network. And before training, I test the pretrain model on the same test set, but only got the 87% accuracy. 

It is werid, anyone helps?

**Comment by tornadomeet** [comment]
it is strange, does your model after load has been changed anywhere?

**Comment by Ldpe2G** [comment]
@tornadomeet , no, and it is strange that after loading the pretrain model if I first just go through the training set without changing the network parameter, then test, and the accuracy became normal again.

**Comment by tornadomeet** [comment]
Does the symbol use bn and fix-gamma=True in it?

**Comment by Ldpe2G** [comment]
@tornadomeet yes

**Comment by tornadomeet** [comment]
@Ldpe2G the newset mxnet(will published as 1.0) is not compatibility with pre-trainined model with bn(fix-gamma=True), please refer to this: https://github.com/dmlc/mxnet/pull/3037

**Comment by Ldpe2G** [comment]
@tornadomeet thanks.
